websocket-compiler-plugin/src/main/java/io/ballerina/stdlib/websocket/plugin/PluginConstants.java

```diff
@@ -77,7 +77,9 @@
         INVALID_RETURN_TYPES_IN_RESOURCE("Invalid return type `{0}` provided for function `{1}`, return type "
                 + "should be a subtype of `{2}`", "WEBSOCKET_104"),
         FUNCTION_NOT_ACCEPTED_BY_THE_SERVICE("Function `{0}` not accepted by the service",
-                "WEBSOCKET_105");
+                "WEBSOCKET_105"),
+        INVALID_LISTENER_INIT_PARAMS("`websocket:ListenerConfiguration` not allowed with `http:Listener` "
```

Comment:
Is not it better to have the diagnostic severity along with the error codes ? [Here](https://github.com/ballerina-platform/module-ballerina-websub/blob/a6e1d85c4524e3bdf63e7b6680bbc5b300c0b38f/websub-compiler-plugin/src/main/java/io/ballerina/stdlib/websub/WebSubDiagnosticCodes.java#L26) you could find a sample.

Reply:
Severity is always `ERROR` in the implementation.